op-chain-ops/genesis/layer_two.go

```diff
@@ -112,5 +112,10 @@ func BuildL2Genesis(config *DeployConfig, l1StartBlock *types.Block) (*core.Gene
 		}
 	}
 
```

Comment:
I would much prefer to have a top level function `setupNetworkUpgrades` or `setupHardforks` and then we move the branching logic that is inside of `setupEcotone` outside of `setupEcotone` into `setupNetworkUpgrades`